packages/server/src/threads/automation.ts

```diff
@@ -100,53 +156,47 @@ class Orchestrator {
   private appId: string
   private automation: Automation
   private emitter: ContextEmitter
-  private context: AutomationContext
-  private job: Job
-  private loopStepOutputs: LoopStep[]
+  private job: AutomationJob
   private stopped: boolean
-  private executionOutput: AutomationResponse
+  private executionOutput: AutomationResults
   private currentUser: UserBindings | undefined
 
   constructor(job: AutomationJob) {
-    let automation = job.data.automation
-    let triggerOutput = job.data.event
-    const metadata = triggerOutput.metadata
-    this.chainCount = metadata ? metadata.automationChainCount! : 0
+    this.automation = job.data.automation
+
+    const triggerOutput = job.data.event
+    if (
```

Comment:
It would be good to explain this, looks a bit random just appearing in the constructor, a comment or a utility function to perform the function of this (I'm guessing this is filling in missing information in a cron trigger if its not supplied)